### PR TITLE
Use Null Coalesce Operator

### PR DESCRIPTION
**Pull request by carusogabriel**
## Type

- Enhancement

### CODE REVIEW

**Comment by carakas** [comment]
I wouldn't really call this an issue, as we clean up the code we refactor it. But I don't think it warrants a separate issue

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Acl/Acl.php [hunk 1]
```diff
@@ -19,28 +19,28 @@
 
 /**
  * The Acl class.
- *
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Acl/Acl.php]: you should revert the changes to everything within the ckfinder folder. It is an external library so we'd prefer to keep it the way it was released

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Backend/Adapter/Dropbox.php [hunk 2]
```diff
@@ -40,7 +40,7 @@ public function __construct(Client $client, array $backendConfig)
     {
         $this->backendConfig = $backendConfig;
 
-        parent::__construct($client, isset($backendConfig['root']) ? $backendConfig['root'] : null);
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Backend/Adapter/Dropbox.php]: idem

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Backend/Backend.php [hunk 3]
```diff
@@ -101,7 +101,7 @@ public function getName()
      */
     public function getTrackedOperations()
     {
-        return isset($this->backendConfig['trackedOperations']) ? $this->backendConfig['trackedOperations'] : array();
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Backend/Backend.php]: idem

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Backend/BackendFactory.php [hunk 4]
```diff
@@ -135,7 +135,7 @@ protected function registerDefaultAdapters()
             $client = S3Client::factory($clientConfig);
 
             $filesystemConfig = array(
-                'visibility' => isset($backendConfig['visibility']) ? $backendConfig['visibility'] : 'private'
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Backend/BackendFactory.php]: idem

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Command/CopyFiles.php [hunk 5]
```diff
@@ -68,7 +68,7 @@ public function execute(Request $request, ResourceTypeFactory $resourceTypeFacto
 
             $copiedFile = new CopiedFile($name, $folder, $resourceType, $this->app);
 
-            $options = isset($arr['options']) ? $arr['options'] : '';
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Command/CopyFiles.php]: idem

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Command/ImageEdit.php [hunk 6]
```diff
@@ -104,7 +104,7 @@ public function execute(Request $request, WorkingFolder $workingFolder, EventDis
                         throw new InvalidRequestException();
                     }
                     $degrees = $actionInfo['angle'];
-                    $bgcolor = isset($actionInfo['bgcolor']) ? $actionInfo['bgcolor'] : 0;
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Command/ImageEdit.php]: idem

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Command/MoveFiles.php [hunk 7]
```diff
@@ -68,7 +68,7 @@ public function execute(Request $request, ResourceTypeFactory $resourceTypeFacto
 
             $movedFile = new MovedFile($name, $folder, $resourceType, $this->app);
 
-            $options = isset($arr['options']) ? $arr['options'] : '';
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Command/MoveFiles.php]: idem

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Command/QuickUpload.php [hunk 8]
```diff
@@ -66,8 +66,8 @@ public function onQuickUploadResponse(FilterResponseEvent $event)
         if ($response instanceof JsonResponse) {
             $responseData = $response->getData();
 
-            $fileUrl = isset($responseData['url']) ? $responseData['url'] : '';
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Command/QuickUpload.php]: idem

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Config.php [hunk 9]
```diff
@@ -222,7 +222,7 @@ protected function mergeDefaultOptions($options)
             'proxyCommand' => 0
         );
 
-        $options['cache'] = array_replace($cacheDefaults, isset($options['cache']) ? $options['cache'] : array());
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Config.php]: idem

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Image.php [hunk 10]
```diff
@@ -168,11 +168,11 @@ public function __construct($imageData, $bmpSupport = false)
             throw new CKFinderException('Unsupported image type');
         }
 
-        $this->width    = isset($imgInfo[0])          ? $imgInfo[0]          : 0;
-        $this->height   = isset($imgInfo[1])          ? $imgInfo[1]          : 0;
-        $this->mime     = isset($imgInfo['mime'])     ? $imgInfo['mime']     : '';
-        $this->bits     = isset($imgInfo['bits'])     ? $imgInfo['bits']     : 8;
-        $this->channels = isset($imgInfo['channels']) ? $imgInfo['channels'] : 3;
+        $this->width    = $imgInfo[0]          ?? 0;
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Image.php]: idem

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/ResourceType/ResourceType.php [hunk 11]
```diff
@@ -78,7 +78,7 @@ public function getDeniedExtensions()
 
     public function getLabel()
     {
-        return isset($this->configNode['label']) ? $this->configNode['label'] : null;
+        return $this->configNode['label'] ?? null;
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/ResourceType/ResourceType.php]: idem

File: src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Translator.php [hunk 12]
```diff
@@ -35,7 +35,7 @@ class Translator
      */
     public function __construct($langCode = null)
     {
-        $locale = $langCode ?: (isset($_GET['langCode']) ? (string) $_GET['langCode'] : 'en');
+        $locale = $langCode ?: ((string) ($_GET['langCode'] ?? 'en'));
```
Comment (carakas) [inline src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Translator.php]: idem

**Comment by carusogabriel** [comment]
@carakas Did it.

**Comment by carakas** [comment]
@carusogabriel Thank you for you contribution 🍰
